### PR TITLE
[1.17] vendor: bump github.com/containers/libpod from v1.8.0 to v1.8.1

### PR DESCRIPTION
**Pull request by mccv1r0**
vendor: bump github.com/containers/libpod from v1.8.0 to v1.8.1

Signed-off-by: Michael Cambria <mcambria@redhat.com>

/kind feature

bump libpod vendor code from v1.8.0 to v1.8.1 to obtain updated libpod and cni features

Fixes None

### CODE REVIEW

**Comment by mccv1r0** [comment]
quai.io issue:

`
Error pulling image quay.io/crio/crio-build-amd64-go1.13:1.17-1.1.1: Error response from daemon: Get https://quay.io/v2/crio/crio-build-amd64-go1.13/manifests/1.17-1.1.1: received unexpected HTTP status: 500 Internal Server Error... retrying
  image cache not found on this host, downloading quay.io/crio/crio-build-amd64-go1.13:1.17-1.1.1
`

I'll try again tomorrow

**Comment by saschagrunert** [comment]
Re-triggered the CircleCI jobs

**Comment by haircommander** [comment]
integration failures seem legit
```
not ok 102 Ensure correct CNI plugin namespace/name/container-id arguments
# (in test file ./network.bats, line 91)
#   `[ "$status" -eq 0 ]' failed
# 0
```
you can run it with `sudo bats test/network.bats`

**Comment by mccv1r0** [comment]
> you can run it with `sudo bats test/network.bats`

I'm getting this:

`$ sudo bats test/network.bats
 ✗ ensure correct hostname
   (from function `setup_crio' in file test/helpers.bash, line 277,                                                                                                               
    from function `start_crio' in file test/helpers.bash, line 340,                                                                                                               
    in test file test/network.bats, line 15)                                                                                                                                      
     `start_crio' failed with status 127                                                                                                                                          
   /home/mcambria/go/src/github.com/cri-o/cri-o/test/helpers.bash: line 277: /home/mcambria/go/src/github.com/cri-o/cri-o/test/copyimg/copyimg: No such file or directory         
   time="2020-03-19T10:06:22-04:00" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"                                                             
   time="2020-03-19T10:06:32-04:00" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"                                                             
   0                        `

Is there something else I need to make?

**Comment by haircommander** [comment]
> > you can run it with `sudo bats test/network.bats`
> 
> I'm getting this:
> 
> `$ sudo bats test/network.bats ✗ ensure correct hostname (from function `setup_crio' in file test/helpers.bash, line 277,
> from function `start_crio' in file test/helpers.bash, line 340, in test file test/network.bats, line 15) `start_crio' failed with status 127
> /home/mcambria/go/src/github.com/cri-o/cri-o/test/helpers.bash: line 277: /home/mcambria/go/src/github.com/cri-o/cri-o/test/copyimg/copyimg: No such file or directory
> time="2020-03-19T10:06:22-04:00" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
> time="2020-03-19T10:06:32-04:00" level=fatal msg="failed to connect: failed to connect: context deadline exceeded"
> 0 `
> 
> Is there something else I need to make?

yes, `make test-binaries` will solve that

**Comment by mccv1r0** [comment]
> yes, `make test-binaries` will solve that

Tests pass on laptop:

`
$ sudo bats test/network.bats
[sudo] password for mcambria: 
 ✓ ensure correct hostname
 ✓ ensure correct hostname for hostnetwork:true
 ✓ Check for valid pod netns CIDR
 ✓ Ensure correct CNI plugin namespace/name/container-id arguments
 ✓ Connect to pod hostport from the host
 ✓ Clean up network if pod sandbox fails
 ✓ Clean up network if pod sandbox fails after plugin success

7 tests, 0 failures

]$ 
`

The hostport kept failing when testing out of my sb due to both wired and wifi adapters being active.  Turning off wifi fixed this.  Wasted hours <g>

File: go.mod [hunk 1]
```diff
@@ -11,20 +11,23 @@ require (
...
-	github.com/containernetworking/cni v0.7.1
-	github.com/containernetworking/plugins v0.8.3
-	github.com/containers/buildah v1.11.5-0.20191031204705-20e92ffe0982
+	github.com/containernetworking/cni v0.7.2-0.20200304161608-4fae32b84921
+	github.com/containernetworking/plugins v0.8.5
+	github.com/containers/buildah v1.14.2
 	github.com/containers/conmon v2.0.11+incompatible
 	github.com/containers/image/v5 v5.2.1
-	github.com/containers/libpod v1.6.3-0.20191101152258-04e8bf3dba50
+	github.com/containers/libpod v1.8.1
```
Comment (TomSweeneyRedHat) [inline go.mod]: 1.8.2 just released yesterday, I'm not sure if you want to bump this now or not.

File: go.mod [hunk 2]
```diff
@@ -11,20 +11,23 @@ require (
...
 	github.com/containers/ocicrypt v0.0.0-20190930154801-b87a4a69c741
-	github.com/containers/psgo v1.4.0 // indirect
-	github.com/containers/storage v1.15.8
+	github.com/containers/storage v1.16.2
```
Comment (TomSweeneyRedHat) [inline go.mod]: This too just bumped to 1.16.5

**Comment by TomSweeneyRedHat** [comment]
You might have a merge dance with https://github.com/cri-o/cri-o/pull/3429

**Pull request merged**